presto-thrift-connector-api/src/main/java/com/facebook/presto/connector/thrift/api/PrestoThriftFunctionId.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.connector.thrift.api;
+
+import com.facebook.drift.annotations.ThriftConstructor;
+import com.facebook.drift.annotations.ThriftField;
+import com.facebook.drift.annotations.ThriftStruct;
+
+import java.util.List;
+import java.util.Objects;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+@ThriftStruct
```

Comment:
@highker I'm adding `PrestoThriftFunctionId` and `PrestoThriftUdfService` directly to thrift connector here because thrift function uses thrift pages etc from thrift-connector. But should we do some refactoring to break this up to `presto-thrift-datatypes`, `presto-thrift-connector-api` and `presto-thrift-udf-api`?